var/spack/repos/builtin/packages/liggghts/package.py

```diff
@@ -89,6 +93,16 @@ def edit(self, spec, prefix):
         if '+profile' in spec:
             makefile.filter(r'^(USE_PROFILE = ).*', r'\1"ON"')
 
+        # Determine how to build libs in LIGGGHTS way
+        if spec.satisfies("libs=shared") and spec.satisfies("libs=static"):
```

Comment:
Since docs are failing and we have to perform a few changes anyhow:
```suggestion
        if spec.satisfies("libs=shared,static"):
```
should be a shorter way to express what you already have.